index.html

```diff
@@ -794,33 +794,7 @@ <h3 class="col-md-11 col-lg-11">Electrify</h3>
 
   <!-- end footer -->
   <script src="./scripts/mobile-pillars.js"></script>
+  <script type="text/javascript" src="./scripts/sticky-nav.js"></script>
```

Comment:
@alexanthony813 I think we no longer need: `type="text/javascript"` anymore, most modern browsers support script tags without it, specially if using html5 doctype.